packages/util-address/types/index.d.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../src/index.js"],"names":[],"mappings":"AAAA;;;;GAIG;AACF,oCAHU,MAAM,GACJ,MAAM,CAKlB;AAED;;;;GAIG;AACH,oCAHW,MAAM,GACJ,MAAM,CAKlB;AAED;;;;GAIG;AACH,iCAHW,MAAM,GACJ,MAAM,CAIlB"}
```

Comment:
Should this be checked in?

Reply:
I was wondering the same thing. Can remove if that's what everyone thinks.